src/DatabaseGpt.SqlServer/SqlServerDatabaseGptProvider.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Data;
+using System.Text;
+using Dapper;
+using DatabaseGpt.Abstractions;
+using DatabaseGpt.Abstractions.Exceptions;
+using Microsoft.Data.SqlClient;
+
+namespace DatabaseGpt.SqlServer;
+
+public class SqlServerDatabaseGptProvider : IDatabaseGptProvider, IDisposable
+{
+    private readonly SqlServerDatabaseGptProviderConfiguration settings;
+    private bool disposedValue;
+
+    public SqlServerDatabaseGptProvider(SqlServerDatabaseGptProviderConfiguration settings)
+    {
+        this.settings = settings;
```

Comment:
The `settings` argument is used only in the constructor, so you don't need to store it in a private field.